.github/workflows/workflow.yml

```diff
@@ -14,7 +14,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@master
+        uses: actions/checkout@v2
+
+      - run: git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*
```

Comment:
@megheaiulian if you put in @v1, it works:
https://github.com/Neovici/cosmoz-frontend/issues/1793

Reply:
Yeah but v2 allows a bit more control over what we fetch during checkout. e.g: fetch only master and the pr branch.

Reply:
Isn't that what v1 does?

Reply:
Went with v1